ui/src/app/devices/views/DeviceDetails/DeviceConfiguration/DeviceConfiguration.test.tsx

```diff
@@ -85,7 +86,11 @@ describe("DeviceConfiguration", () => {
       </Provider>
     );
 
-    userEvent.click(screen.getByRole("button", { name: Label.Edit }));
+    userEvent.click(
+      screen.getAllByRole("button", {
+        name: EditableSectionLabels.EditButton,
+      })[0]
+    );
```

Comment:
This is admittedly a code smell, since `EditableSection` renders two identical buttons and hides one of them with a utility class depending on the viewport size. This could be fixed by using a viewport size listener, but that seems a bit heavy. Or alternatively using a custom CSS grid, but that gets a bit tricky trying to line it up with Vanilla's default grid (especially when it's nested). I'm open to suggestions.

Reply:
I think it's an OK trade-off. At least the button code is not being repeated and it's using standard vanilla classes so it's easy to understand what's going on.